components/brave_wallet/common/common_utils_unittest.cc

```diff
@@ -842,13 +842,13 @@ TEST_F(BraveWalletPolicyTest, PolicyDisablesWallet) {
   // Test that the policy preference is set correctly
   EXPECT_TRUE(prefs_.GetBoolean(prefs::kDisabledByPolicy));
 
-#if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC) || BUILDFLAG(IS_LINUX)
```

Comment:
cc @supermassive had to update the `BraveWalletPolicyTest` unit test, since Android is now the only platform not to support, I've flipped the BUILDFLAG check